test_fms/run_test.sh

```diff
@@ -0,0 +1,35 @@
+# Function to set up and run tests. 
```

Comment:
There is already a test_common.sh for this purpose. No need to add another file. Just add this to test_common.sh.in. You can even use variable substitution from the configure step.